airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json

```diff
@@ -64,37 +65,44 @@
       },
       "client_id": {
         "type": "string",
-        "description": "ID of your Microsoft Advertising client application.",
+        "title": "Client ID",
+        "description": "The Client ID of your Microsoft Advertising developer application.",
         "airbyte_secret": true
       },
       "client_secret": {
         "type": "string",
-        "description": "Secret of your Microsoft Advertising client application.",
+        "title": "Client Secret",
+        "description": "The Client Secret of your Microsoft Advertising developer application.",
         "airbyte_secret": true
       },
       "customer_id": {
         "type": "string",
-        "description": "User's customer ID."
+        "title": "Customer ID",
+        "description": "User's Customer ID."
       },
       "developer_token": {
         "type": "string",
+        "title": "Developer Token",
         "description": "Developer token associated with user.",
         "airbyte_secret": true
       },
       "refresh_token": {
         "type": "string",
-        "description": "The long-lived Refresh token received via grant_type=refresh_token request.",
+        "title": "Refresh Token",
+        "description": "Refresh Token to renew the expired Access Token.",
         "airbyte_secret": true
       },
       "user_id": {
         "type": "string",
+        "title": "User ID",
         "description": "Unique user identifier."
```

Comment:
what is the difference between this and customer ID? where can the user obtain this ID?

Reply:
It seems like user ID is Account ID. But not quite sure